ddl/ddl.go

```diff
@@ -413,11 +411,9 @@ func (d *ddl) genGlobalID() (int64, error) {
 
 // generalWorker returns the first worker. The ddl structure has only one worker before we implement the parallel worker.
 // It's used for testing.
+// TODO: Remove this function.
```

Comment:
Why remove it?

Reply:
Because it's only used in tests, and we can use `d.workers[generalWorker]` directly.